pkgs/development/python-modules/tinygrad/default.nix

```diff
@@ -62,29 +67,20 @@ buildPythonPackage rec {
       substituteInPlace tinygrad/runtime/autogen/opencl.py \
         --replace-fail "ctypes.util.find_library('OpenCL')" "'${ocl-icd}/lib/libOpenCL.so'"
     ''
-    # hipGetDevicePropertiesR0600 is a symbol from rocm-6. We are currently at rocm-5.
-    # We are not sure that this works. Remove when rocm gets updated to version 6.
     + lib.optionalString rocmSupport ''
-      substituteInPlace extra/hip_gpu_driver/hip_ioctl.py \
-        --replace-fail "processor = platform.processor()" "processor = ${stdenv.hostPlatform.linuxArch}"
```

Comment:
Was this also related to rocm6 or why are we removing this?

Reply:
This file is within `extra` and now we don't really care about `extra` anymore, so I removed it.
I could bring it back though.

Reply:
Idk what extra was and why we cared before but I trust you jaja